client/src/www/app/main.electron.ts

```diff
@@ -127,6 +127,21 @@ class ElectronErrorReporter implements OutlineErrorReporter {
   }
 }
 
+class ElectronMethodChannel implements MethodChannel {
+  async invokeMethod(methodName: string, params: string): Promise<string> {
+    switch (methodName) {
+      default:
+        return await window.electron.methodChannel.invoke(
```

Comment:
Maybe the `await` (and the `async` above) is not needed here? IIRC, `methodChannel.invoke` should already return a `Promise<string>`.

Reply:
I think it's clearer if we call await. Otherwise it reads as if we are returning a promise of a promise, even though Typescript auto-unwraps it.

Reply:
I would not recommend `return invoke(...)` in an `async function`. Instead, I would maybe prefer `return invoke(...)` in a regular `function`, since we are not doing any multiple asynchronous steps here.

Reply:
Done. But I also removed the switch to avoid mistakes. Previously I was implementing an interception there, and I needed the await.